packages/appsync-modelgen-plugin/src/utils/process-connections-v2.ts

```diff
@@ -43,12 +50,20 @@ export function getConnectedFieldV2(field: CodeGenField, model: CodeGenModel, co
     }
 
     // when there is a fields argument in the connection
-    const connectedFieldName = indexDirective ? ((fieldDir: CodeGenFieldDirective) => { return fieldDir.fieldName ;})(indexDirective as CodeGenFieldDirective) : DEFAULT_HASH_KEY_FIELD;
+    const connectedFieldName = indexDirective
+      ? ((fieldDir: CodeGenFieldDirective) => {
+          return fieldDir.fieldName;
+        })(indexDirective as CodeGenFieldDirective)
+      : DEFAULT_HASH_KEY_FIELD;
 
     // Find a field on the other side which connected by a @connection and has the same fields[0] as indexName field
     const otherSideConnectedField = connectedModel.fields.find(f => {
       return f.directives.find(d => {
-        return (d.name === 'belongsTo' || d.name === 'hasOne' || d.name === 'hasMany') && d.arguments.fields && d.arguments.fields[0] === connectedFieldName;
+        return (
+          (d.name === 'belongsTo' || d.name === 'hasOne' || d.name === 'hasMany') &&
+          d.arguments.fields &&
+          d.arguments.fields[0] === connectedFieldName
```

Comment:
These checks can be simplified with optional chaining.

Reply:
these are just a commit hook for styling, not actual changes here, but I can update